### ISSUE TITLE
Add copyright license 

### ISSUE THREAD

**Issue by wendtke** [SmogDr/edar_coursebook#7]
A [Creative Commons](https://creativecommons.org/licenses/) license? Most open source `bookdown` courses use CC. You can use the ["license chooser" ](https://creativecommons.org/choose/) to help decide. [CSU Ventures](https://csuventures.org/licensing-available-technologies/) also has opinions on this, as I am sure you know. They don't support or defend open source licenses, but, if you are planning on selling this course to a MOOC platform like Coursera, you might want to talk to CSU Ventures first.

**Comment by SmogDr** [comment]
Let’s use the same one as Brooke. Since we used her material we should be consistent with her license.

jv

--
John Volckens
Professor, Mechanical Engineering
Colorado State University
he, him, his
jv.colostate.edu<http://jv.colostate.edu/> / @Smogdr

On Jul 12, 2020, at 2:44 PM, Kathleen Wendt <notifications@github.com> wrote:

﻿

A Creative Commons<https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcreativecommons.org%2Flicenses%2F&data=02%7C01%7Cjohn.volckens%40colostate.edu%7Cea1ca8072dcb480376ce08d826a46785%7Cafb58802ff7a4bb1ab21367ff2ecfc8b%7C0%7C1%7C637301834848825031&sdata=3FCnBIuvjx62XBE9c6R7%2Bm7dI3LzeFI8UmPfWmEAuKo%3D&reserved=0> license? Most open source bookdown courses use CC.

"License chooser" <https://nam01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcreativecommons.org%2Fchoose%2F&data=02%7C01%7Cjohn.volckens%40colostate.edu%7Cea1ca8072dcb480376ce08d826a46785%7Cafb58802ff7a4bb1ab21367ff2ecfc8b%7C0%7C1%7C637301834848835027&sdata=K8EewnuxWGjMZYE5ki%2BbYPS%2FBMOjvH77Regkf2sL4ms%3D&reserved=0>

**Comment by wendtke** [comment]
She used [CC 4.0](https://creativecommons.org/licenses/by-nc/4.0/)

**Comment by wendtke** [comment]
Need to add license before posting [fivethirtyeight twitter data](https://github.com/fivethirtyeight/data/tree/master/twitter-ratio)?

**Comment by SmogDr** [comment]
Yes – will you add in the same license as Brooke’s course?  That seems the safest for now.

--
John Volckens
Professor, Mechanical Engineering
Colorado State University
he, him, his
jv.colostate.edu / @Smogdr
